core/templates/pages/exploration-editor-page/exploration-save-and-publish-buttons/exploration-save-and-publish-buttons.component.ts

```diff
@@ -66,7 +71,28 @@ angular.module('oppia').component('explorationSaveAndPublishButtons', {
         ExplorationSaveService.discardChanges();
       };
 
+      let isModalDisplayed = false;
+
       $scope.getChangeListLength = function() {
+        var countChanges = ChangeListService.getChangeList().length;
+
+        if (countChanges >= 50 && !isModalDisplayed &&
```

Comment:
Let's create a named constant for `50` so it is clear what it represents to a reader.

Reply:
DONE